components/homepage/WorkExp.tsx

```diff
@@ -58,10 +58,26 @@ const WorkExp = (props: Props) => {
           </div>
         </div>
         <div className="flex flex-col gap-[1rem] md:ml-[1.62rem] md:gap-[2.19rem]">
-          <WorkExpCard />
-          <WorkExpCard />
-          <WorkExpCard />
-          <WorkExpCard />
+          <WorkExpCard
+            title={"Slack"}
+            body={"Product Design - 4 Years Experince"}
+            image={"/Slack.svg"}
+          />
+          <WorkExpCard
+            title={"Firefox"}
+            body={"Sr. UI Design - 4 Years Experince"}
+            image={"Firefox.svg"}
+          />
+          <WorkExpCard
+            title={"Digital Ocean"}
+            body={"UI & Motion Designer - 4 Years Experince"}
+            image={"Ocean.svg"}
+          />
+          <WorkExpCard
+            title={"Mailchimp"}
+            body={"UI Designer - 4 Years Experince"}
+            image={"Chimp.svg"}
+          />
```

Comment:
Same idea. If your work experience changes it would be nice to just update sanity instead of having to open your project, edit the source code, rebuild and then deploy

Reply:
@brandonetter I originally planed on using sanity for just the blog. This makes a lot of sense though, with being able to update without changing the project. I'm currently working on finishing the rest of the pages and still have to figure out how to add in Sanity and use it. Should I have this merged and then go back and add everything to json when I incorporate Sanity or focus on Sanity now?